portfolio/src/main/webapp/index.html

```diff
@@ -50,6 +50,15 @@ <h1>My Portfolio</h1>
         <button onclick="window.location.href='books.html';"/>Books</button>
         <button onclick="window.location.href='projects.html';"/>My Projects</button>
         <button onclick="getRandomQuoteUsingArrowFunctions()"/>Fun Fact!</button>
+        <div id='fun-fact'></div>
```

Comment:
Use double-quotes instead of single-quotes for attributes. ([Style Guide link](https://google.github.io/styleguide/htmlcssguide.html#HTML_Quotation_Marks))
More important than the particular style, though, is consistency. If you use double-quotes for something, use double-quotes for all other instances of that same thing, and vice versa.
```suggestion
        <div id="fun-fact"></div>
```